lib/src/zap/zap.c

```diff
@@ -1010,8 +1010,8 @@ static void init_atfork(void)
 	assert(zev_queue);
 
 	for (i = 0; i < zap_event_workers; i++) {
-		char thread_name[16];
-		(void)snprintf(thread_name, 16, "zap:worker:%d", i);
+		char thread_name[64];
+		(void)snprintf(thread_name, 64, "zap:worker:%d", i);
```

Comment:
A pthread name can't be bigger than 16B (see pthread_setname_np). Probably just clamp the %d to a width of 4 and I think it will go away.

Reply:
Oh. Thanks for pointing that out. I'll fix it now.